internal/blocktx/processor.go

```diff
@@ -57,6 +58,7 @@ type Processor struct {
 	registerTxsBatchSize        int
 	registerRequestTxsBatchSize int
 	tracingEnabled              bool
+	attributes                  []attribute.KeyValue
```

Comment:
Nit:
Will this field be used only for tracing?

If so, maybe a better name would be something like `tracingAttributes`?